docs/power-payment-app.md

```diff
@@ -0,0 +1,127 @@
+# Power Payment App
+
+## Overview
+
+**What does it do?**
+
+PowerPay is a mobile payment application that allows users to send and receive money, top-up their accounts, and withdraw cash. It offers a flexible payment experience by utilizing both smartphones and physical devices called Choronko.
+
+**Who is using it?**
+
+- PowerPay Users: Individuals with smartphones who can initiate and manage transactions through the app.
+- PowerPay Kiosks: Physical kiosks that facilitate top-ups and withdrawals for users with smartphones and Choronko.
+- Choronko Users: Individuals who can send and receive payments using the Choronko device without needing a smartphone.
+
+## Technologies used
+
+- ReactJS for building the user interface and implementing a Progressive Web App (PWA) approach.
+- HTML and CSS for the structure and styling of the user interface.
+- RESTful APIs for communication with the backend server.
+
+## Architecture and Interaction
+
+### User Interface:
+
+- **Registration Process**: The registration process is a series of steps that allows users to create an account and access the payment app's features. The steps typically include:
+
+    1. **Step 1 - Personal Information**: Users enter their personal details, such as name, phone number etc. This information is necessary for account creation and verification.
+
+    2. **Step 2 - Pin Setup**: Users set up a secure pin to protect their account. The pin should meet specific criteria, such as minimum length of numbers etc.
+
+    3. **Step 3 - OTP Confirmation**: Once users have entered their information, set up a pin, they are provide with an OTP in which they now confirm their account for security measures.
```

Comment:
It was discussed, that this was the other way around. A user enters a phone number alongside a name, and the system send a OTP. Then that one is used with the PUN to ensure the user phone really exists, without exposing the combination `phoneNumber`+`PIN`.

Reply:
Understood sir..

ok so here, i simple just need to interchange the flow ensuring that the user is provided with th OTP before setting his/her pin....